dexmaker-mockito-inline build fails with Cannot notify listeners of type ProjectEvaluationListener as these listeners are already being notified.

This only happens when syncing gradle files from inside Android Studio. ./gradlew build works. Downgrading to org.jfrog.buildinfo:build-info-extractor-gradle:4.3.0 causes other failures.

This seems to be https://github.com/JFrogDev/build-info/issues/86

---

Error:FAILURE: Build failed with an exception.

* Where:
Build file '/usr/local/google/home/moltmann/projects/dexmaker/dexmaker-mockito-inline/build.gradle' line: 66

* What went wrong:
A problem occurred configuring project ':dexmaker-mockito-inline'.
> Failed to notify project evaluation listener.
   > A problem occurred configuring project ':dexmaker'.
      > Cannot notify listeners of type ProjectEvaluationListener as these listeners are already being notified.
   > A problem occurred configuring project ':dexmaker'.
      > Cannot notify listeners of type ProjectEvaluationListener as these listeners are already being notified.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 0s

---

@drewhannay Any idea how we can work around this?

--- a/gradle.properties
+++ b/gradle.properties
@@ -15,6 +15,9 @@
 org.gradle.jvmargs=-Djava.awt.headless=true
 org.gradle.parallel=true
 
+# Disabling due to https://github.com/JFrogDev/build-info/issues/86
+org.gradle.configureondemand=false
+
 # When configured, Gradle will run in incubating parallel mode.
 # This option should only be used with decoupled projects. More details, visit
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects

